posthog-core/src/index.ts

```diff
@@ -44,7 +44,7 @@ class PostHogFetchNetworkError extends Error {
 }
 
 function isPostHogFetchError(err: any): boolean {
-  return typeof err === 'object' && (err.name === 'PostHogFetchHttpError' || err.name === 'PostHogFetchNetworkError')
+  return typeof err === 'object' && (err instanceof PostHogFetchHttpError || err instanceof PostHogFetchNetworkError)
```

Comment:
@benjackwhite was there a reason the checks were not typesafe instead?

Reply:
the checks could fail if the code is aggressively minified

Reply:
This was an external contribution iirc so 🤷

Reply:
true https://github.com/PostHog/posthog-js-lite/pull/88/